web/samples/index.html

```diff
@@ -20,7 +20,7 @@
   </head>
   <body>
   <h1>KeymanWeb Samples</h1>
-  <h2><a href="./uncompiled.html">Test uncompiled Keymanweb</a></h2>
+  <h2><a href="./uncompiled.html">Test unminified Keymanweb</a></h2>
```

Comment:
If `uncompiled` is now `unminified`, should we go ahead and rename `uncompiled.html` to `unminified.html`?  
(along with any other pages referencing uncompiled.html in samples/ and testing/)